<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterForm</class>
 <widget class="QDialog" name="RegisterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="Title">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Firstname</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="Firstname">
     <property name="styleSheet">
      <string notr="true">border-radius:4px;
padding:4px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Surname</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="Surname">
     <property name="styleSheet">
      <string notr="true">border-radius:4px;
padding:4px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="Email">
     <property name="styleSheet">
      <string notr="true">border-radius:4px;
padding:4px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="Password">
     <property name="styleSheet">
      <string notr="true">border-radius:4px;
padding:4px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Confirm Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ConfirmPass">
     <property name="styleSheet">
      <string notr="true">border-radius:4px;
padding:4px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Pin</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="PinEntry">
     <property name="styleSheet">
      <string notr="true">border-radius:4px;
padding:4px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="RegisterBtn">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(5, 109, 120);
color: rgb(255, 255, 255);
border-radius:8px;
padding-top:4px;
padding-bottom:4px;
padding-left:25px;
padding-right:25px;</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="AlreadyHaveAcc">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(227, 0, 114);
color: rgb(255, 255, 255);
border-radius:8px;
padding-top:4px;
padding-bottom:4px;
padding-left:25px;
padding-right:25px;</string>
     </property>
     <property name="text">
      <string>Already got an account?</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
